modules/caddyhttp/logging.go

```diff
@@ -58,8 +58,11 @@ type ServerLogConfig struct {
 }
 
 // wrapLogger wraps logger in a logger named according to user preferences for the given host.
-func (slc ServerLogConfig) wrapLogger(logger *zap.Logger, host string) *zap.Logger {
-	if loggerName := slc.getLoggerName(host); loggerName != "" {
+func (slc ServerLogConfig) wrapLogger(logger *zap.Logger, req *http.Request) *zap.Logger {
```

Comment:
Yeah I agree that appending would go against the purpose of this feature, so overriding seems reasonable. We can wait until your PR gets merged and discuss how we can make `access_logger_name` take multiple values so that it fits in the multiple logger feature addition in your PR.

Reply:
@francislavoie I went the naive route by making `access_logger_names` take multiple values seperated by a `,`. Do you recall what was your idea here and if we should rather implement it? I was thinking about tweaking the logic of the `UnmarshalCaddyFile` function of the `VarsMiddleware` but I wanted to get your opinion first.